src/solver/variable/state.h

```diff
@@ -97,7 +111,17 @@ class State
     **
     ** \param areaWideIndex Index of the thermal cluster for the current area
     */
-    void yearEndBuildFromThermalClusterIndex(const unsigned int areaWideIndex, uint numSpace);
+    void yearEndBuildFromThermalClusterIndex(const unsigned int areaWideIndex);
+
+    void yearEndBuildThermalClusterCalculateStartupCosts(const uint& maxDurationON,
```

Comment:
You should split state.h into 2 headers
- state.h for all public functions
- state_priv.h for all private functions, which we don't really want to expose outside of state.cpp